srcs/response/Response.cpp

```diff
@@ -132,13 +133,38 @@ OutgoingData * Response::handleGet(const t_server & server, const HTTPRequest &
 OutgoingData * Response::handlePost(const t_server & server, const HTTPRequest & req)
 {
 	(void)server;
-	(void)req;
-	return NULL;
+	std::string uri = req.getUriWithoutQString();
+	const t_route *routeptr = getRouteFromUri(server, uri);
+	if (routeptr == NULL) {
+		std::cout << "     no route found from uri: " << uri << std::endl;
+		return makeResponse(404, "Not Found", "text/plain", "404 Not Found");
+	}
+	const t_route &route = *routeptr;
```

Comment:
Why do this and not use the pointer you already have?

Reply:
ask aurelien

Reply:
Can you use your brain before implementing logic please? We expect better at our corporation, this is not CrowdStrike.

We will talk about it tomorrow during your standup.